Foundation/FileManager.swift

```diff
@@ -2377,46 +2377,107 @@ extension FileManager {
             self.innerEnumerator = ie
         }
         
-        @available(Windows, deprecated, message: "Not Yet Implemented")
         override func nextObject() -> Any? {
             let o = innerEnumerator.nextObject()
             guard let url = o as? URL else {
                 return nil
             }
 
 #if os(Windows)
-            NSUnimplemented()
+            let path = url.path.replacingOccurrences(of: baseURL.path+"\\", with: "")
 #else
             let path = url.path.replacingOccurrences(of: baseURL.path+"/", with: "")
+#endif
             _currentItemPath = path
             return path
-#endif
         }
     }
 
-#if os(Windows)
     internal class NSURLDirectoryEnumerator : DirectoryEnumerator {
-        internal typealias ErrorHandler = /* @escaping */ (URL, Error) -> Bool
+#if os(Windows)
+        var _options : FileManager.DirectoryEnumerationOptions
+        var _errorHandler : ((URL, Error) -> Bool)?
+        var _stack: [URL]
+        var _current: URL?
```

Comment:
Can we name this `_url` instead?  That way we can make `_options`, `_errorHandler`, and `_url` common to all the implementations.

Reply:
`_current` here more aligns with what `_current` is doing in the Unix implementation: storing the most recently visited file. While `url` in the Unix implementation is saving the url parameter. I think it would be a bit confusing to use them in different ways.

Reply:
Bleh, didn't see `_current` there an thought that it was the `_url` equivalent.